Formula/asdf.rb

```diff
@@ -0,0 +1,32 @@
+class Asdf < Formula
+  desc "Extendable version manager with support for Ruby, Node.js, Erlang & more"
+  homepage "https://github.com/asdf-vm"
+  url "https://github.com/asdf-vm/asdf/archive/v0.3.0.tar.gz"
+  sha256 "1d986789bfcbef76dc23cc968ca192a03862a9496df65e7cd5d6f8117bc85789"
+
+  bottle :unneeded
+
+  depends_on "coreutils" => :run
```

Comment:
These don't all have to be `:run`; that's only used for dependencies that are normally build-time-only, like automake, autoconf, and libtool.